docs/architecture/security/principles/01-servers-are-not-trusted.mdx

```diff
@@ -0,0 +1,37 @@
+# P01 - Servers are not trusted
+
+Clients do not need to trust Bitwarden server or web infrastructure to be good actors in supplying
+data to keep an account's encrypted content secure. In other words, the special status of providing
+a network sync for client data does not enable a server, or any entity between the server and
+client, to take any action that would aid in decryption of a user's vault by the server.
+
+This is what we mean when we sometimes refer to "End-to-end encrypted."
+
+## Account key sharing as a feature
+
+This principle does not mean that clear text data are never shared, but rather that any such
+exposure is always declared to the user and exclusively between accounts, never to the server or
```

Comment:
suggestion: consider using the existing phrase "Informed and explicit consent" instead of "declared to the user"

Reply:
Nice. See ec84aafd922ba937f0d96613815255ed027eedbe